ledger/blockqueue.go

```diff
@@ -143,6 +143,11 @@ func (bq *blockQueue) syncer() {
 			bq.mu.Unlock()
 
 			minToSave := bq.l.notifyCommit(committed)
+			// Save one extra block in the queue for checking timestamps
```

Comment:
Set min to one round earlier to save the extra block

Reply:
You wont need this if the txtail would return -1  on committedupto